src/index.html

```diff
@@ -14,8 +14,44 @@
     }
   </style>
 
-  <body>
+  <body >
```

Comment:
Please, remove this content of the body.

Reply:
resolved